features/step_definitions/file_steps.coffee

```diff
@@ -4,18 +4,22 @@ path = require 'path'
 
 module.exports = ->
 
-  @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+  @Given /^my project has no test files$/, (done) ->
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^my project has a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
In regards to choosing a particular style because somebodies Git tool or text editor make it hard to change indented code or read diffs, that argument is fine for private projects. For professional/open-source projects, however, we should aim for the most readable style. Variable names only change very few times (typically less than 3 over their whole lifetime), while they are read several thousand times in such projects. The needs of the many outweigh the needs of the few here.   :)